docs/pages/docs/guides/custom-admin-ui-navigation.md

```diff
@@ -292,13 +307,50 @@ Otherwise, all lists will be added.
 ```tsx
 const CustomNavigation = ({ lists }) => (
   <NavigationContainer>
-    <ListNavItems lists={lists} />
+    <NavItemGroup>
+      <ListNavItems lists={lists} />
+    </NavItemGroup>
   </NavigationContainer>
 )
 ```
 
 ![example of navigation without include prop values](/assets/guides/custom-admin-ui-navigation/listNavItems-without-include.png)
 
+### ListNavItem
+
+This component will render a single `NavItem` for the given list.
+
+```tsx
+import { ListNavItem } from '@keystone-6/core/admin-ui/components'
+```
+
+In this example we create groups for our lists.
+
+```tsx
+const listGroups = [
+  [
+    { name: 'People', lists: ['User', 'Bio', 'Role']},
+    { name: 'Posts', lists: ['Post', 'Category', 'Tag']},
+  ]
+];
+
+const CustomNavigation = ({ lists }) => (
+  <NavigationContainer>
+    {listGroups.map(group => (
+      <Box key={group.name}>
+        <H5 paddingX="xlarge">{group.name}</H5>
+        <NavItemGroup>
+          {group.lists.map((key) => {
+            const list = lists.find((l) => l.key === key);
+            return list ? <ListNavItem key={key} list={list} /> : null;
+          })}
+        </NavItemGroup>
+      </Box>
+    ))}
+  </NavigationContainer>
+)
+```
```

Comment:
Kind of contrived, you could do this with the `ListNavItems` component too but I wanted to show that this component is also exported.